todo-with-kuzzle-and-vueJS/src/views/KuzzleConnect.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-app light id="KuzzleConnect">
+    <v-container grid-list-sm text-xs-center>
+      <div class="center" id="circle">
+        <v-progress-circular :size="100" :width="10" color="blue" indeterminate></v-progress-circular>
+      </div>
+    </v-container>
+  </v-app>
+</template>
+
+<script>
+import kuzzle from '../service/Kuzzle.js';
+
+export default {
+  name: 'KuzzleConnect',
+  data() {
+    return {
+      indexName: 'todolists'
+    };
+  },
+  methods: {
+    async connect() {
+      try {
+        await kuzzle.connect();
+        clearInterval(this.interval);
+        const exists = await kuzzle.index.exists(this.indexName);
+        if (!exists) {
+          await kuzzle.index.create(this.indexName);
+          const mapping = {
+            properties: {
+              complete: { type: 'boolean' },
+              task: { type: 'text' }
+            }
+          };
+          await kuzzle.collection.create(this.indexName, 'FirstList', mapping);
+        }
+        localStorage.setItem('indexName', this.indexName);
+        localStorage.setItem('connectedToKuzzle', true);
```

Comment:
I think this state should not live in the local storage. This is Kuzzle SDK internal state and you should track it using listeners on the instance. You can keep a value in some root component or such, but why in the Local Storage?